dipy/viz/actor.py

```diff
@@ -0,0 +1,648 @@
+        plane_colors.SetInputConnection(image_resliced.GetOutputPort())
+        plane_colors.Update()
+        image_actor.input_connection(plane_colors)
+    else:
+        image_actor.input_connection(image_resliced)
+    image_actor.display()
+    image_actor.opacity(opacity)
+
+    return image_actor
+
+
+def streamtube(lines, colors=None, opacity=1, linewidth=0.01, tube_sides=9,
+               lod=True, lod_points=10 ** 4, lod_points_size=3,
+               spline_subdiv=None, lookup_colormap=None):
+    """ Uses streamtubes to visualize polylines
+
+    Parameters
+    ----------
+    lines : list
+        list of N curves represented as 2D ndarrays
+
+    colors : array (N, 3), list of arrays, tuple (3,), array (K,), None
+        If None then a standard orientation colormap is used for every line.
+        If one tuple of color is used. Then all streamlines will have the same
+        colour.
+        If an array (N, 3) is given, where N is equal to the number of lines.
+        Then every line is coloured with a different RGB color.
+        If a list of RGB arrays is given then every point of every line takes
+        a different color.
+        If an array (K, ) is given, where K is the number of points of all
+        lines then these are considered as the values to be used by the
+        colormap.
+        If an array (L, ) is given, where L is the number of streamlines then
+        these are considered as the values to be used by the colormap per
+        streamline.
+        If an array (X, Y, Z) or (X, Y, Z, 3) is given then the values for the
+        colormap are interpolated automatically using trilinear interpolation.
+
+    opacity : float
+    linewidth : float
+    tube_sides : int
+    lod : bool
+        use vtkLODActor rather than vtkActor
+    lod_points : int
+        number of points to be used when LOD is in effect
+    lod_points_size : int
+        size of points when lod is in effect
+    spline_subdiv : int
+        number of splines subdivision to smooth streamtubes
+    lookup_colormap : bool
+        add a default lookup table to the colormap
+
+    Examples
+    --------
+    >>> from dipy.viz import fvtk
+    >>> r=fvtk.ren()
+    >>> lines=[np.random.rand(10, 3), np.random.rand(20, 3)]
+    >>> colors=np.random.rand(2, 3)
+    >>> c=fvtk.streamtube(lines, colors)
+    >>> fvtk.add(r,c)
+    >>> #fvtk.show(r)
+
+    Notes
+    -----
+    Streamtubes can be heavy on GPU when loading many streamlines and
+    therefore, you may experience slow rendering time depending on system GPU.
+    A solution to this problem is to reduce the number of points in each
+    streamline. In Dipy we provide an algorithm that will reduce the number of
+    points on the straighter parts of the streamline but keep more points on
+    the curvier parts. This can be used in the following way
+
+    from dipy.tracking.distances import approx_polygon_track
+    lines = [approx_polygon_track(line, 0.2) for line in lines]
```

Comment:
Maybe: 

```
... can be used in the following way::
   
    from dipy.tracking.distances import approx_polygon_track
```
etc.

This will probably make this render better in sphinx.